### PR TITLE
fix: statistic token

### PR DESCRIPTION
**Pull request by MadCcc**
- [x] Component: Button
- [x] Component: Affix  
- [x] Component: Divider  
- [x] Component: Empty  
- [x] Component: Typography  
- [x] Component: Carousel  
- [x] Component: Checkbox   
- [x] Component: Select  
- [x] Component: Tree  
- [x] Component: TreeSelect  
- [ ] ~~Component: AutoComplete~~
- [x] Component: Cascader  
- [x] Component: Slider  
- [x] Component: Input 
- [x] Component: InputNumber  
- [x] Component: Mentions  
- [x] Component: Form  
- [x] Component: Alert
- [x] Component: Popover
- [x] Component: Avatar 
- [x] Component: Tag 
- [x] Component: Result 
- [x] Component: Badge
- [x] Component: Statistic  
- [x] Component: Tabs   
- [x] Component: Rate  
- [x] Component: Switch  
- [x] Component: Pagination  
- [x] Component: Skeleton  
- [x] Component: Grid  🚀  
- [x] Component: Spin  🚀  
- [x] Component: Descriptions 🚀  
- [x] Component: Drawer   🚀  
- [x] Component: List  
- [x] Component: Image 🚀  
---

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | -          |
| 🇨🇳 Chinese | -          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/affix/style/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // deps-lint-skip-all
 import { CSSObject } from '@ant-design/cssinjs';
-import { GenerateStyle, genComponentStyleHook, FullToken } from '../../_util/theme';
+import { GenerateStyle, genComponentStyleHook, FullToken, mergeToken } from '../../_util/theme';
```
Comment (afc163) [inline components/affix/style/index.tsx:3]: mergeToken 是做什么用的？

Comment (MadCcc) [reply components/affix/style/index.tsx:3]: 处理类似 `{ ...token, xxxToken: xxx }` 的逻辑，但是不用解构语法。token 统计改了 getter，解构语法会把所有 token 都统计进去。

Comment (afc163) [reply components/affix/style/index.tsx:3]: 不应该为了开发工具来修改源码写法，这样不可持续。

Comment (MadCcc) [reply components/affix/style/index.tsx:3]: 也许可以让解构语法报错？目前看 Proxy 会有一点侵入，但是时比较有效的方法。

Comment (zombieJ) [reply components/affix/style/index.tsx:3]: 可以搞个 lint 在 styles 的文件里禁止直接解构，proxy 这个侵入性更大了。不适合。

Comment (zombieJ) [reply components/affix/style/index.tsx:3]: lint 额外搞个 PR 弄，这里只做清理。

**Pull request merged**